scripts/src/routes/index.ts

```diff
@@ -38,17 +38,25 @@ function proxyOverRouter(router: express.Router, proxy: ExtendedRouter, obj: any
 
 const AUTHENTICATED_METHODS = ["get", "delete", "post", "put", "patch"];
 
+enum AuthScopes {
+	tos = 1,
+}
```

Comment:
added the concept of permission scopes

Reply:
does it matter what is the ordinal for the enums?
why did you go with ` = 1` instead of just doing: `enum AuthScopes { tos }`?
also, i prefer that enum value names start with a capital, and in this case the entire string:
```
enum AuthScopes {
	TOS = 1,
}
```